python/lsst/resources/http.py

```diff
@@ -27,6 +27,7 @@
 from typing import TYPE_CHECKING, BinaryIO, Iterator, List, Optional, Tuple, Union, cast
 
 import requests
+from astropy import units as u
```

Comment:
This requires that we add `astropy` to the `https` optional dependencies section in `pyproject.toml`.